<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Задача коммивояжер - генетический алгоритм</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>11</y>
      <width>188</width>
      <height>253</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblGenerations">
       <property name="text">
        <string>Число поколений</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leGenerationsNum">
       <property name="text">
        <string>800</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblMutationProbability">
       <property name="text">
        <string>Вероятность мутации</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leMutationProbability">
       <property name="text">
        <string>0.03</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPopulationSize">
       <property name="text">
        <string>Размер популяции</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePopulationSize">
       <property name="text">
        <string>320</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSolve">
       <property name="text">
        <string>Решить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPathLength">
       <property name="text">
        <string>Длина найденного маршрута</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePathLength">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>961</width>
      <height>225</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="teLog"/>
     </item>
     <item>
      <widget class="QProgressBar" name="pbSolutionFound">
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
